webapp/Connector/src/view/PageHeader.js

```diff
@@ -134,7 +134,7 @@ Ext.define('Connector.view.PageHeader', {
             flex: 2
         };
 
-        var detailSearchField = Ext.create('Ext.form.field.Text', {
+        var detailSearchField = Ext.create('Connector.view.Learn.SearchField', {
```

Comment:
I did this to standardize the search field for all the learn pages, the custom control will render the clear (x) icon when there is search filter present.